src/Controller/Adminhtml/Page/Save.php

```diff
@@ -141,7 +141,7 @@ protected function hydrateLandingPage(LandingPageInterface $landingPage, array $
             $data[LandingPageInterface::OVERVIEW_PAGE_ID] = null;
         }
 
-        $landingPage->setUrlPath($data[LandingPageInterface::URL_PATH]);
+        $landingPage->setUrlPath(trim($data[LandingPageInterface::URL_PATH]));
```

Comment:
I would suggest to do this in `setUrlPath` method itself, than it's fixed wherever the source of the save is.
i.e. API, some import mechanism etc. etc.